### ISSUE TITLE
USER STORY:  User Profile

### ISSUE THREAD

**Issue by DeeMcCart** [DeeMcCart/CI_PP5_Jeweller#5]
EPIC:  #1 

As a **user** I can **personalise my profile** so that **I can take advantage of greater site functionality**
(In fact there will be multiple user stories which depend on this particular task as a pre-requisite)

### Assumptions or Pre-Requisites:
*  EPIC #1 is underway
* Issue #3 has been done
* Issue #4 has been done (this takes care of the basic register-login-logout functionality)

### Acceptance Criteria: (Must be completed before task is moved to 'Done')
- [ ] Profile personalisation page exists
- [ ] Includes newsletter signup checkbox (and current setting) - only if user has supplied email address
- [ ] Includes a section/link for wishlists (doesnt have to be an active section yet?)
- [ ] Includes a section/link for orders (doesnt have to be an active section yet?) with payment status and delivery status.
- [ ] Includes ability to upload profile avatar
- [ ] Avatar displays at top of page in navbar - success message Welcome XXXX

### Tasks

- [x] Create Profile app, models.py, views.py, urls.py, admin.py, profile.html
- [x] Build and apply Migrations
- [x] Verify the Profile data structure by adding in admin console.
- [ ] Create user profile page
- [ ] Customise it with sections identified in acceptance criteria above
- [ ] Task3
- [ ] Task4

Before changing task status to 'Review' or 'Done' please provide comment (and screenprints if appropriate) as documentary evidence of task completion

**Comment by DeeMcCart** [comment]
profiles app installed 23/01/24
profiles/models.py: UserProfile and UserAddress models created.
LucidChart created/ updated with database model to reflect

**Comment by DeeMcCart** [comment]
29/01/23 Closing out this issue with incomplete elements for Sprint1 - will create a separate issue for remaining items for Sprint2
